### PR TITLE
Feature cluster reporting results #189

### PR DESCRIPTION
**Pull request by lkolacz**
##### Feature list
* https://projects.invisionapp.com/share/EDBIM0Y8M#/screens/243476913

##### Progress checker
- [x] Django

- [x] Polymer

- [x] Django test cases

- [ ] ~Polymer test cases~

### CODE REVIEW

**Comment by rkrupinski** [comment]
@ErinOC @n1207n I'm aware of the code redundancy among async dropdown-based filters. We can address that in a separate issue, if time permits.

**Comment by n1207n** [comment]
I'm taking a stab for fixing cluster indicator list API. I suspect maybe fake data generation issue where response plan is not properly linked.

**Comment by n1207n** [comment]
I fixed to see cluster indicator objects finally on UI. I think something happened to fake data (for the most part) and Added more ORM filtering in API side to target other cluster indicator types.

Choose first response plan in cluster portal and you should be able to see some indicators.

File: polymer/src/elements/cluster-reporting/cluster-report-list.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                mode="[[mode]]">
+            </cluster-report-proxy>
+          </div>
+        </etools-data-table-row>
+      </template>
+
+      <etools-loading active="[[loading]]"></etools-loading>
+    </div>
+
+    <etools-data-table-footer
```
Comment (rkrupinski) [inline polymer/src/elements/cluster-reporting/cluster-report-list.html]: Spec mentions pagination.

**Comment by rkrupinski** [comment]
@ErinOC Once you reset the database, do migrations & generate fake data, use response plan `240` (simply alter the url) in order to get some data back.

**Pull request merged**